README.adoc

```diff
@@ -51,7 +51,49 @@ You can **run the https://github.com/rust-lang/rust-clippy[Clippy linter]** usin
 
 == Working in Omicron
 
-Omicron is a pretty large repo containing a bunch of related components.  The repo is structured as one large Cargo workspace.  When you run `cargo build` or `cargo test`, you'll wind up building and testing everything.  In practice, people are usually only working on one or two components at a time.  You can usually iterate faster by specifying a specific package, like `cargo build -p omicron-nexus`.
+Omicron is a pretty large repo containing a bunch of related components.  If you just build the whole thing with `cargo build` or `cargo test`, it can take a while, even for incremental builds.  Since most people are only working on a few of these components at a time, it's helpful to be know about Cargo's tools for working with individual packages in a workspace.
+
+NOTE: This section assumes you're already familiar with the prerequisites and environment setup needed to do _any_ work on Omicron.  See xref:docs/how-to-run-simulated.adoc[] or xref:docs/how-to-run.adoc[] for more on that.
+
+Here's a summary:
```

Comment:
Could we use a heading or some other visual indicator here to make it more clear that this is the summary?